packages/mdl-snackbar/README.md

```diff
@@ -0,0 +1,145 @@
+
+```javascript
+import MDLSnackbar from 'mdl-snackbar';
+
+const snackbar = new MDLSnackbar(document.querySelector('.mdl-snackbar'));
+```
+
+### Showing a message and action
+
+Once you have obtained an MDLSnackbar instance attached to the DOM, you can use
+the `show` method to trigger the display of a message with optional action. The
+`show`  method takes an object for snackbar data. The table below shows the
+properties and their usage.
+
+| Property | Effect | Remarks | Type |
+|-----------|--------|---------|---------|
+| message   | The text message to display. | Required | String |
+| timeout   | The amount of time in milliseconds to show the snackbar. | Optional (default 2750) | Integer |
+| actionHandler | The function to execute when the action is clicked. | Optional | Function |
+| actionText | The text to display for the action button. | Required if actionHandler is set |  String |
+| multiline | Whether to show the snackbar with space for multiple lines of text | Optional |  Boolean |
+| actionOnBottom | Whether to show the action below the multiple lines of text | Optional, applies when multiline is true |  Boolean |
+
+### Using the Foundation Class
+
+MDL Snackbar ships with an `MDLSnackbarFoundation` class that external frameworks and libraries can
+use to integrate the component. As with all foundation classes, an adapter object must be provided.
+The adapter for snackbars must provide the following functions, with correct signatures:
+
+| Method Signature | Description |
+| --- | --- |
+| `addClass(className: string) => void` | Adds a class to the root element. |
+| `removeClass(className: string) => void` | Removes a class from the root element. |
+| `setAriaHidden() => void` | Sets `aria-hidden="true"` on the root element. |
+| `unsetAriaHidden() => void` | Removes the `aria-hidden` attribute from the root element. |
+| `setMessageText(message: string) => void` | Set the text content of the message element. |
+| `setActionText(actionText: string) => void` | Set the text content of the action element. |
+| `setActionAriaHidden() => void` | Sets `aria-hidden="true"` on the action element. |
+| `unsetActionAriaHidden() => void` | Removes the `aria-hidden` attribute from the action element. |
+| `registerActionClickHandler(handler: EventListener) => void` | Registers an event handler to be called when a `click` event is triggered on the action element. |
+| `deregisterActionClickHandler(handler: EventListener) => void` | Deregisters an event handler from a `click` event on the action element. This will only be called with handlers that have previously been passed to `registerActionClickHandler` calls. |
+| `registerTransitionEndHandler(handler: EventListener) => void` | Registers an event handler to be called when an `transitionend` event is triggered on the root element. Note that you must account for
+vendor prefixes in order for this to work correctly. |
```

Comment:
could you remove the line break here? Otherwise it breaks to a new row in the table.